contracts/Controller.sol

```diff
@@ -60,10 +60,17 @@ contract Controller is Initializable, OwnableUpgradeSafe, ReentrancyGuardUpgrade
     /// @dev mapping between an account owner and their approved or unapproved account operators
     mapping(address => mapping(address => bool)) internal operators;
 
+    /******************************************************************** V2.0.0 storage upgrade ******************************************************/
```

Comment:
how do other projects do this?

Reply:
Maybe /*** text ***/ ? https://github.com/compound-finance/compound-protocol/blob/master/contracts/ComptrollerG6.sol#L92 like this?

Reply:
Not sure stylistically what the best way of separating it is

Reply:
Not sure if it is important, just to make sure it is clear that all those new variables are in order for storage slot